exchanges/hitbtc/hitbtc.go

```diff
@@ -211,7 +211,7 @@ func (e *Exchange) GetBalances(ctx context.Context) (map[string]Balance, error)
 		url.Values{},
 		otherRequests,
 		&result)
-	ret := make(map[string]Balance)
+	ret := make(map[currency.Code]Balance)
```

Comment:
can you please move this below the err check? Or even just change the func signature to return []Balance